cli/cli/kurtosis_config/resolved_config/kurtosis_cluster_config.go

```diff
@@ -73,11 +81,27 @@ func NewKurtosisClusterConfigFromOverrides(clusterId string, overrides *v4.Kurto
 		}
 	}
 
+	var grafloki GrafanaLoki
+	if overrides.GraflokiConfig != nil {
+		grafloki = GrafanaLoki{
+			ShouldStartBeforeEngine: overrides.GraflokiConfig.ShouldStartBeforeEngine,
+			GrafanaImage:            overrides.GraflokiConfig.GrafanaImage,
+			LokiImage:               overrides.GraflokiConfig.LokiImage,
+		}
+	}
+
+	shouldTurnOffDefaultLogsSink := defaultShouldTurnOffDefaultLogsSink
```

Comment:
1. Same mention about negatives vs positives
2. Ohhh wait, so `defaultShouldTurnOffDefaultLogsSink` is used here as well as above?? (EDIT: wait no it's not... we seemingly have the `defaultShouldTurnOffDefaultLogsSink` constant, and we also have a variable called `defaultShouldTurnOffDefaultLogsSink` that I commented on earlier. Do these need to be unified?)